geoutils/georaster.py

```diff
@@ -69,6 +69,31 @@ def _resampling_from_str(resampling: str) -> Resampling:
     return resampling_method
 
 
+# Function to set the default nodata values for any given dtype
+# Similar to GDAL for int types, but without absurdly long nodata values for floats.
+# For unsigned types, the maximum value is chosen (with a max of 99999).
+# For signed types, the minimum value is chosen (with a min of -99999).
+def default_ndv(dtype: str) -> int:
+    """
+    Set the default nodata value for any given dtype, when this is not provided.
+    """
+    default_ndv_lookup = {
+        "uint8": 255,
+        "int8": -128,
+        "uint16": 65535,
+        "int16": -32768,
+        "uint32": 99999,
+        "int32": -99999,
+        "float32": -99999,
```

Comment:
Isn't the norm `-9999`? Not saying it's better, I'm just considering consistency.

Reply:
It was just weird to have a nodata value that was smaller (in absolute) than the nodata value for int16 for example. And I've seen this value used when I was searching for default GDAL nodata values, but I can't find out where anymore.
Anyway, we could change that easily if needed.